Makefile

```diff
@@ -42,10 +42,11 @@ FUSES      = -U hfuse:w:0xd8:m -U lfuse:w:0xff:m
 # Tune the lines below only if you know what you are doing:
 
 AVRDUDE = avrdude  $(PROGRAMMER) -p $(DEVICE) -B 10 -F -D
-COMPILE = avr-gcc -Wall -Os -DF_CPU=$(CLOCK) -mmcu=$(DEVICE) -I. -ffunction-sections
+COMPILE = avr-gcc  -Wall -Os -g -DF_CPU=$(CLOCK) -mmcu=$(DEVICE) -I. -ffunction-sections
```

Comment:
You could create a new compile line here, maybe called DEBUG, that adds the -g flag